barretenberg/cpp/src/barretenberg/crypto/merkle_tree/lmdb_store/lmdb_tree_store.test.cpp

```diff
@@ -258,372 +256,372 @@ TEST_F(LMDBTreeStoreTest, can_write_and_read_leaves_by_hash)
-    {
-        LMDBTreeWriteTransaction::Ptr transaction = store.create_write_transaction();
-        for (uint32_t i = 0; i < values.size(); i++) {
-            store.write_leaf_key_by_index(values[i], i + leafIndexStart, *transaction);
-        }
-        transaction->commit();
-    }
-
-    {
-        LMDBTreeWriteTransaction::Ptr transaction = store.create_write_transaction();
-        store.delete_all_leaf_keys_before_or_equal_index(deleteFromIndex, *transaction);
-        transaction->commit();
-    }
-
-    {
-        LMDBTreeReadTransaction::Ptr transaction = store.create_read_transaction();
-        std::vector<bb::fr> leafKeys;
-        store.read_all_leaf_keys_before_or_equal_index(leafIndexStart + 1023, leafKeys, *transaction);
-        EXPECT_EQ(leafKeys.size(), 1024);
-        for (uint32_t i = 0; i < leafKeys.size(); i++) {
-            EXPECT_EQ(leafKeys[i], values[1023 - i]);
-        }
-    }
-
-    {
-        LMDBTreeReadTransaction::Ptr transaction = store.create_read_transaction();
-        std::vector<bb::fr> leafKeys;
-        store.read_all_leaf_keys_before_or_equal_index(2000, leafKeys, *transaction);
-        EXPECT_EQ(leafKeys.size(), 1024);
-        for (uint32_t i = 0; i < leafKeys.size(); i++) {
-            EXPECT_EQ(leafKeys[i], values[1023 - i]);
-        }
-    }
-    {
-        LMDBTreeReadTransaction::Ptr transaction = store.create_read_transaction();
-        std::vector<bb::fr> leafKeys;
-        store.read_all_leaf_keys_before_or_equal_index(10, leafKeys, *transaction);
-        EXPECT_EQ(leafKeys.size(), 0);
-    }
-}
-
-TEST_F(LMDBTreeStoreTest, can_retrieve_all_keys_when_none_are_present)
-{
-    std::vector<bb::fr> values = create_values(1024);
-    LMDBTreeStore store(_directory, "DB1", _mapSize, _maxReaders);
-
-    {
-        LMDBTreeReadTransaction::Ptr transaction = store.create_read_transaction();
-        std::vector<bb::fr> leafKeys;
-        store.read_all_leaf_keys_after_or_equal_index(0, leafKeys, *transaction);
-        EXPECT_EQ(leafKeys.size(), 0);
-    }
-
-    {
-        LMDBTreeReadTransaction::Ptr transaction = store.create_read_transaction();
-        std::vector<bb::fr> leafKeys;
-        store.read_all_leaf_keys_before_or_equal_index(0, leafKeys, *transaction);
-        EXPECT_EQ(leafKeys.size(), 0);
-    }
-}
-
-TEST_F(LMDBTreeStoreTest, can_delete_all_keys_when_none_are_present)
-{
-    std::vector<bb::fr> values = create_values(1024);
-    LMDBTreeStore store(_directory, "DB1", _mapSize, _maxReaders);
-
-    {
-        LMDBTreeWriteTransaction::Ptr transaction = store.create_write_transaction();
-        store.delete_all_leaf_keys_after_or_equal_index(0, *transaction);
-        transaction->commit();
-    }
-
-    {
-        LMDBTreeWriteTransaction::Ptr transaction = store.create_write_transaction();
-        store.delete_all_leaf_keys_before_or_equal_index(0, *transaction);
-        transaction->commit();
-    }
-}
+// TEST_F(LMDBTreeStoreTest, can_read_write_key_by_index)
```

Comment:
Why all the commented-out tests? Let's reenable or delete them.

Reply:
Yes, they are no longer relevant